Utils/secrets.py

```diff
@@ -0,0 +1,109 @@
+""" fetch secrets by key from appropriate source according to environment """
+
+from enum import Enum
+import os
+
+
+TEAM_PREFIXES = {"HIVE", "BST"}
+PROJECT_IDS = {
+    "DEV": "amazing-city-422521-t1",
```

Comment:
<img width="548" alt="image" src="https://github.com/user-attachments/assets/a297eacc-fa61-4e80-9155-955f9026047b">
Here are the project ids. I think the dev project depends on what team. If Hive do 01 if BST do 00.